app/index.html

```diff
@@ -82,6 +82,7 @@
     <script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
     <script src="bower_components/lodash/lodash.js"></script>
     <script src="bower_components/bootstrap/dist/js/bootstrap.js"></script>
+    <script src="bower_components/bootstrap-combobox/js/bootstrap-combobox.js"></script>
```

Comment:
@spadgett, no, I'm not using bootstrap-combobox, but it's part of PatternFly, so it's getting pulled in accordingly.  We can exclude it if you think it's a problem.